src/components/navBar/index.tsx

```diff
@@ -1,17 +1,68 @@
 import Image from 'next/image';
 import styles from '@/components/navBar/navBar.module.scss';
-
+import { useEffect, useState } from 'react';
+import Link from 'next/link';
 const RDSLogo = '/RDSLogo.png';
+const edit = '/pencil.webp';
+
+function debounce<A = unknown, R = void>(
+  fn: (args: A) => R,
+  ms: number
+): (args: A) => Promise<R> {
+  let timer: NodeJS.Timeout;
```

Comment:
Should we have a `NodeJS` type in NextJS?

Reply:
I couldn't find anything specific to nextjs in timer type